apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1/project.json

```diff
@@ -0,0 +1,43 @@
+﻿{
+  "version": "0.1.0-*",
+  "description": "Google Compute Engine Metadata v1 client library",
```

Comment:
Although we have a general format for our library descriptions, it doesn't really fit in here anyway as there isn't an API this is really wrapping (other than compute), so don't worry about that part.

I'd be tempted to remove the v1 part from the description though, as "Compute Engine Metadata" isn't really a dedicated API.